.github/workflows/bounty-payout.yml

```
@@ -0,0 +1,97 @@
+        if: steps.check-issues.outputs.has-bounty == 'true'
+        env:
+          ISSUE_NUMBER: ${{ steps.check-issues.outputs.issue-number }}
+          BOUNTY_AMOUNT: ${{ steps.check-issues.outputs.bounty-amount }}
+          CONTRIBUTOR_USERNAME: ${{ steps.check-issues.outputs.contributor-username }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          BLT_API_TOKEN: ${{ secrets.BLT_API_TOKEN }}
+          API_BASE_URL: ${{ secrets.API_BASE_URL || 'https://owasp.org' }}
+        uses: actions/github-script@v6.1.1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.7)

key "github-token" is duplicated in "with" section. previously defined at line:83,col:11. note that this key is case insensitive

(syntax-check)